package.json

```diff
@@ -35,8 +35,9 @@
     "css-loader": "~0.23.1",
     "del": "~2.2.0",
     "edx-pattern-library": "~0.12.5",
-    "eslint": "~2.13.1",
-    "eslint-config-edx": "~1.2.1",
+    "eslint": "^3.16.0",
```

Comment:
Yep, we do need it since Yarn won't link binaries for transitive deps and recommends either an explicit dependency or a postinstall creating the link manually. The explicit dependency seemed like a cleaner option in this case.